api/src/main/resources/application.yaml

```diff
@@ -178,4 +178,5 @@ server:
     threads:
       worker: 128
       io: 16
+  #port: ${HTTP_PORT}
```

Comment:
Is this needed @kamal-mohammed ? Can it be removed?

Reply:
Yes, it is needed. We have another util coming that uses this as a placeholder.